index.html

```diff
@@ -2,11 +2,229 @@
+              <label for="advanced">Advanced</label>
+            </li>
+          </ul>
+  
+          <h3>Lives</h3>
+          <ul class="flex-row game-options-list">
+            <li>
+              <input type="radio" id="oneLife" name="livesSelect" value="1">
+              <label for="oneLife">
+                <i class="fas fa-cat"></i>
+              </label>
+            </li>
+            <li>
+              <input type="radio" id="threeLives" name="livesSelect" value="3">
+              <label for="threeLives">
+                <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i>
+              </label>
+            </li>
+            <li>
+              <input type="radio" id="fiveLives" name="livesSelect" value="5">
+              <label for="fiveLives">
+                <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i>
+              </label>
+            </li>
+            <li>
+              <input type="radio" id="sevenLives" name="livesSelect" value="7">
+              <label for="sevenLives">
+                <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i>
+              </label>
+            </li>
+            <li>
+              <input type="radio" id="nineLives" name="livesSelect" value="9">
+              <label for="nineLives">
+                <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i>
+              </label>
+            </li>
+          </ul>
+  
+          <input type="submit" value="Begin!">
+        </form>
+      </div>
+  
+      <div class="flex-center flex-column game-play" id="gamePlay">
+        <div class="flex-center flex-row game-status">
+          <ul class="flex-center flex-column lives-count">
+            <li><h3>Lives</h3></li>
+            <li id="livesCount"><i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i> <i class="fas fa-cat"></i></li>
+          </ul>
+          <h2 class="game-prompt" id="gamePrompt">Multiples of 2</h2>
+          <ul class="flex-center flex-column game-score">
+            <li><h3>Score</h3></li>
+            <li id="gameScore"><h4>100456</h4></li>
+          </ul>
+        </div>
+        <div class="player" id="player"><i class="fas fa-cat"></i></div>
+        <div class="enemy" id="enemy"><i class="fas fa-dog"></i></div>
+        <div class="flex-center flex-column game-board">
+          <ul class="flex-center flex-row game-row" id="row-5">
```

Comment:
This works as a template. Once the elements are being added to the DOM with javascript, we should think about using index 0 - 29, which is the same as the arrays for our game logic. It would be more difficult to transpose cell-4-4 to index 9 of the array for example.

Reply:
Okay, you are welcome to remove and replace and restyle as you see fit. You obviously are pretty set in how you want to do this.